src/utility/symmetrization.jl

```diff
@@ -68,8 +68,18 @@ function _fit_spaces(
     end
     return y
 end
+
+function ChainRulesCore.rrule(::typeof(_fit_spaces), y::AbstractTensorMap, x::AbstractTensorMap)
+    function pullback(Δ)
+        return NoTangent(), _fit_spaces(Δ, y), NoTangent()
+    end
+    return _fit_spaces(y, x), pullback
+end
```

Comment:
I'm not entirely convinced that this is correct, could you test something like this using a finitedifference approach?

Reply:
Yes, I tested it.